### ISSUE TITLE
Cannot create symlink: freqstart/docker/user_data

### ISSUE THREAD

**Issue by lsiem** [berndhofer/freqstart#77]
When running the script with the setup routine, I get the following error message:

```
+ Freqtrade volume setup:

+ Run project: freqtrade.yml
  Image installed: freqtradeorg/freqtrade:stable
Creating network "freqtrade_default" with the default driver
Creating freqtrade_freqtrade_run ... done
  Directory created: /user_data
! [WARNING] Do NOT manually change the permissions of the Freqtrade volume!
sudo: cd: command not found
sudo: "cd" is a shell built-in command, it cannot be run directly.
sudo: the -s option may be used to run a privileged shell.
sudo: the -D option may be used to run a command in a specific directory.

! [ERROR] Cannot create symlink: /home/_/freqstart/docker/user_data
```

**Comment by berndhofer** [comment]
Oh god, im sorry. I kinda "fixed" a working function to pass a maybe possible error with relative paths by creating symlinks. Will revert it to v2.0.0 version.

**Comment by tomjrtsmith** [comment]
I got the same

**Comment by tomjrtsmith** [comment]
Hey Bernd, I am still getting the symlink error.

If I try a git pull or git fetch all it says all up to date, and my github
desktop is saying last update was 2 days ago ' Merge pull request #76
<https://github.com/berndhofer/freqstart/issues/76> from berndhofer/develop
' yet I'm fairly certain earlier this morning (NZT) when I looked there was
an update around the symlink error which I read over.

I'm not sure if it's git playing up or whether you might have rolled back
some changes?

**Comment by berndhofer** [comment]
Reverted the function to use abspath again. Fixed in v2.1.2
